LICENSE.txt

```diff
@@ -193,27 +176,44 @@ solely pursuant to the terms of the relevant commercial agreement.
 
    END OF TERMS AND CONDITIONS
 
-   APPENDIX: How to apply the Apache License to your work.
+===============================================================================
+
+This section describes all third-party components bundled with the CrateDB
+source code.
+
+As a general guideline, it is a policy of the Apache Software Foundation that
+all bundled third-party components should be available under a category A
+license (https://www.apache.org/legal/resolved.html#category-a).
+
+The "server" directory contains source code that originates from the Apache
+Lucene and Elasticsearch projects.
+
+-------------------------------------------------------------------------------
+
+Elasticsearch
+Copyright 2009-2021 Elasticsearch B.V.
+
+Licensed under the Apache License, Version 2.0 (see above for full license
+text).
+
+Elasticsearch bundles a number of third-party components. For a full list of
+the respective licenses, consult the relevant LICENSE.txt files included with
+the source code.
+
```

Comment:
I don't think this fits here. The license file in this repo is for the contents in this repository. E.g. the Dockerfile definitions and support scripts. 

The license is not for the software within the docker image. 

What's the motivation to change this?